### PR TITLE
style: resizing font-size of events page

### PR DESCRIPTION
**Pull request by Prerna290**
## Type
- ( ) Fix
- ( ) Refactor
- (X) Style
- ( ) Docs
- ( ) Feat
- ( ) Hotfix

### CODE REVIEW

File: apps/commudle-admin/src/app/app-shared-components/event-card/event-card.component.html [hunk 1]
```diff
@@ -24,7 +24,14 @@
                 <nb-icon icon="checkmark-circle-2" nbTooltip="Completed" status="success" class="com-m-auto"></nb-icon>
               </div>
               <div *ngIf="event.interested_members_count > 0" class="interested-members">
-                <span>Interested Members: {{ event.interested_members_count }}</span>
+                <app-badge
+                  [text]="event.interested_members_count"
+                  [color]="'#F0F2F8'"
```
Comment (arpansac) [inline apps/commudle-admin/src/app/app-shared-components/event-card/event-card.component.html:29]: Is there a way to pass the color name and then use it from tailwind in the app-badge component?

File: apps/commudle-admin/src/app/app-shared-components/event-card/event-card.component.html [hunk 2]
```diff
@@ -24,7 +24,14 @@
...
+                  [nbIcon]="'people-outline'"
+                  [fontColor]="'#667085'"
```
Comment (arpansac) [inline apps/commudle-admin/src/app/app-shared-components/event-card/event-card.component.html:31]: Same as above

File: apps/commudle-admin/src/app/app-shared-components/event-card/event-card.component.html [hunk 3]
```diff
@@ -24,7 +24,14 @@
...
+                  [fontSize]="'small'"
+                  [borderRadius]="'24px'"
```
Comment (arpansac) [inline apps/commudle-admin/src/app/app-shared-components/event-card/event-card.component.html:33]: This should be standard, small, medium, large, round edges, round corners, etc. Let's not pass numbers into it

File: apps/commudle-admin/src/app/app-shared-components/event-horizontal-card/event-horizontal-card.component.html [hunk 4]
```diff
@@ -23,15 +23,19 @@
...
-                  <nb-icon icon="people-outline" class="people-outline"></nb-icon>
-                  <span>{{ event.interested_members_count }}</span>
-                </button>
+                <app-badge
+                  [text]="event.interested_members_count"
+                  [color]="'#F0F2F8'"
+                  [nbIcon]="'people-outline'"
+                  [fontColor]="'#667085'"
+                  [fontSize]="'small'"
+                  [borderRadius]="'24px'"
```
Comment (arpansac) [inline apps/commudle-admin/src/app/app-shared-components/event-horizontal-card/event-horizontal-card.component.html:37]: Same as above

File: apps/commudle-admin/src/app/app-shared-components/tech-sessions-card/tech-sessions-card.component.scss [hunk 5]
```diff
@@ -1,39 +1,35 @@
...
 
-    .event-time {
-      @apply com-text-xs com-font-normal com-mb-4 com-mt-0 com-text-Auro-Metal-Saurus;
-    }
-
-    .event-title {
-      @apply com-text-base com-font-semibold com-m-0 com-text-Dark-Jungle-Green md:com-line-clamp-2 com-line-clamp-1;
+      .details {
+        .event-title {
+          @apply md:com-text-sm com-text-base com-font-normal com-m-0 com-text-Dark-Jungle-Green com-line-clamp-1 com-mb-2;
```
Comment (arpansac) [inline apps/commudle-admin/src/app/app-shared-components/tech-sessions-card/tech-sessions-card.component.scss:24]: Is the title color green?

File: apps/commudle-admin/src/app/feature-modules/public-community/components/membership-toggle/membership-toggle.component.html [hunk 6]
```diff
@@ -8,6 +8,7 @@
   [size]="size"
   [appearance]="appearance"
   [ngClass]="{ fullWidth: isMobileWidthFull }"
+  class="join-button"
```
Comment (arpansac) [inline apps/commudle-admin/src/app/feature-modules/public-community/components/membership-toggle/membership-toggle.component.html:11]: class should be join-community

whenever you're accessing this in code, you could do button.join-community

File: tailwind.preset.js [hunk 7]
```diff
@@ -63,6 +63,7 @@ module.exports = {
       },
       gridColumn: {
         'span-16': 'span 16 / span 16',
+        'span-17': 'span 17 / span 17',
```
Comment (arpansac) [inline tailwind.preset.js]: This is for?

File: apps/shared-modules/mini-user-profile/mini-user-profile.module.ts [hunk 8]
```diff
@@ -10,7 +10,7 @@ import { UserProfileCardSmallComponent } from './components/profile-cards/user-p
 import { UserFollowComponent } from './components/user-follow/user-follow.component';
 import { MiniUserProfileDirective } from './directives/mini-user-profile.directive';
 import { HiringLookingWorksTagsComponent } from './components/hiring-looking-works-tags/hiring-looking-works-tags.component';
-
+// import { SharedComponentsModule } from '../../shared-components/shared-components.module';
```
Comment (ArshdeepGrover) [inline apps/shared-modules/mini-user-profile/mini-user-profile.module.ts:13]: Let's make import path relative

File: apps/shared-modules/mini-user-profile/components/profile-cards/user-profile-card-large/user-profile-card-large.component.html
Comment (ArshdeepGrover) [inline apps/shared-modules/mini-user-profile/components/profile-cards/user-profile-card-large/user-profile-card-large.component.html:1]: let's make badge component standalone

**Review by ArshdeepGrover — CHANGES_REQUESTED** [review]

**Pull request merged**